### ISSUE TITLE
The enumerator explanation text doesn't meet a11y contrast guidelines

### ISSUE THREAD

**Issue by reemaka** [civiform/civiform#4990]
**Describe the bug**
The text in blue here isn't contrasted enough:
![image](https://github.com/civiform/civiform/assets/7339672/ff646667-df7e-47f7-9dc3-25154a85d762)

**To Reproduce**

1. Create a new enumerator question.
2. Click "Question enumerator" > "enumerator"
3. Text that shows up in the box at the top of the page is in a low-contrast blue.

**Comment by bion** [comment]
We do not have a11y standards for the admin UI
